modules/apps/user-associated-data/user-associated-data-web/src/main/resources/content/Language_fr.properties

```diff
@@ -12,7 +12,7 @@ are-you-sure-you-want-to-anonymize-the-users-personal-data=Voulez-vous vraiment
 are-you-sure-you-want-to-anonymize-this-entity=Voulez-vous vraiment rendre anonyme cette entité ?
 are-you-sure-you-want-to-deactivate-the-user=Voulez-vous vraiment désactiver l'utilisateur ?
 are-you-sure-you-want-to-delete-the-entities-of-this-application=Êtes-vous sûr de vouloir supprimer les entités de cette application ?
-are-you-sure-you-want-to-delete-the-selected-items=Voulez-vous vraiment supprimer les éléments sélectionnés ?
+are-you-sure-you-want-to-delete-the-selected-items=Voulez-vous vraiment supprimer les éléments sélectionnés ?
```

Comment:
Unsure if `0xa0` (non-breaking space) is critical here, to avoid breaking lines.